modules/gql/queries/allCurrentVotes.ts

```diff
@@ -1,15 +1,34 @@
 import { gql } from 'graphql-request';
 
 export const allCurrentVotes = gql`
```

Comment:
The query in the gov polling db needed to be updated to accept the query variables such as `first`, and this query matches it. See the PR here https://github.com/makerdao/gov-polling-db/pull/127/files